### ISSUE TITLE
Discuss support for ISO8601DateTimeObject validation / formatting

### ISSUE THREAD

**Issue by zikalino** [Azure/magic-module-specs#38]
We should decide where that should be handled.
I am planning to add comparison function for DateTime in generic idempotence check.
I wonder if I should assume that the format was already prechecked / standardized.
Do you think this should be handled by Magic Modules?

**Comment by JunyiYi** [comment]
Nope, MM has not done any validations for these types yet. So feel free to do it in idempotency check function.
